src/Auth0.OidcClient.WPF/WebViewBrowser.cs

```diff
@@ -52,7 +52,7 @@ public Task<BrowserResult> InvokeAsync(BrowserOptions options, CancellationToken
             var webView = new WebViewCompatible();
             window.Content = webView;
 
-            webView.NavigationStarting += (sender, e) =>
+            webView.NavigationCompleted += (sender, e) =>
```

Comment:
Could you explain why this has to be changed from `NavigationStarting` to `NavigationCompleted` ?

Reply:
To properly close the `WebView` control, we must do it on `NavigationCompleted`.

Reply:
I don't think I fully understand the reasoning. When I use NavigationStarting or NavigationCompleted, in both situations the process (Desktop App Web Viewer) is killed correctly. NavigationStarting kicks in before NavigationCompleted, as we are not interested in rendering the contents of that page, I think it makes sense to stick to NavigationStarting. Could you test it with NavigationStarting and see if works for you as well? If it does, we should rollback this change.

Reply:
I added a commit to revert this.